styles/style.css

```diff
@@ -61,6 +59,33 @@ body {
 }
 
 .card-thumbnail {
-  max-height: 250px;
-  overflow: hidden;
+    width: 100%;
+    height: 17vw;
+    border-bottom: 2px solid rgba(0, 0, 0, .26);
 }
+
+.card-thumbnail img {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+
+}
+
+card-body {
+    height: 31vh;
+}
+
+.card-body .card-title {
+    border-bottom: 2px solid rgba(0, 0, 0, .26);
+    height: 11vh;
+}
+
+.card-body .card-text {
+    height: 15vh;
+}
+
+.link-btn {
```

Comment:
Might be worth moving this into another div in the cards if possible, just as it sits on top of some of the text

Reply:
Although I've just realised this is just cos I have an awfully small screen lol so it's squeezing it all together